samples/client/petstore/rust/README.md

```diff
@@ -0,0 +1,97 @@
+# Go API client for swagger
```

Comment:
Good catch. I'll do a grep for "Go" before release to make sure we don't have any Go-related code/doc left over in the Rust generator and templates.